examples/discord_to_desktop_notification.yaml

```
@@ -0,0 +1,44 @@
+apiVersion: v1
+kind: IOConfig
+module: mewbot.io.discord
+name: DiscordIO
+uuid: 1
+properties:
+  token: "[token goes here]"
+
+---
+
+apiVersion: v1
+kind: IOConfig
+module: mewbot.io.desktop_notification
+name: DesktopNotificationIO
+uuid: 2
+properties:
+  token: "[should-not-be_needed]"
```

You might be able to make the properties problem go away with
```suggestion
properties: {}
```
(unless you have already tried that).

---

I would have been prepared to swear I did ... but it would seem not!